test/services/testsvc.jl

```diff
@@ -22,42 +23,42 @@ end
 close(channel::TestRpcChannel) = close(channel.sock)
 
 type SvcHeader
-    method::ASCIIString
+    method::String
```

Comment:
Yes. thanks.
Should be fixed in https://github.com/tanmaykm/ProtoBuf.jl/pull/85.
But maybe we can go ahead with the tagging?